frontend/src/components/electrification/project/ProjectIntakeForm.vue

```diff
@@ -132,7 +132,11 @@ async function onRegisteredNameInput(e: AutoCompleteCompleteEvent) {
     const results = (await externalApiService.searchOrgBook(e.query))?.data?.results ?? [];
     orgBookOptions.value = results
       .filter((x: { [key: string]: string }) => x.type === 'name')
-      .map((x: { [key: string]: string }) => x?.value);
+      // map value and topic_source_id for AutoComplete display and selection
```

Comment:
I know you didn't write most of this function but should match the two you added. `x` -> `obo`